# SURF Notebook 2018

### Author: Hannah Chu
#### Affiliation: University of Florida & John Jay College of Criminal Justice - CUNY


#### Date started: 2018-05-29
#### Date end: 2018-08-04
## Projects & Descriptions
* Summer Undergraduate Research at Florida (SURF) Program
	*  Rhagoletis Viability Selection Project
	*  Proteostasis Project
	*  ECB Biological Rhythms Project

# Table of contents
* [Page 1:   2018-05-29](#id-section1). First Day, tasks, and potential summer projects
* [Page 2:   2018-05-30](#id-section2). Rhagoletis Viability Selection Project Overview
* [Page 3: 2018-05-30 ](#id-section3).Kostal paper - critical reading notes
* [Page 4: 2018-05-31 ](#id-section4).  R coding & ECB rearing; Dopman paper - introduction notes
* [Page 5: 2018-06-01 ](#id-section5). More R coding & new project plan
* [Page 6: 2018-06-03 ](#id-section6). G Matrix & ECB Exp. Plan
* [Page 7: 2018-06-04 ](#id-section7). Survival Data Analysis
* [Page 8:  2018-06-05](#id-section8). Proteome Stability Programming
* [Page 9: 2018-06-06 ](#id-section9). Proteome Stability Project To Do List
* [Page 10: 2018-06-07 ](#id-section10). Water Bath Tests
* [Page 11: 2018-06-08 ](#id-section11). Water Bath Tests II
* [Page 12: 2018-06-10 ](#id-section12). Rhagoletis Viability Selection Analysis -- unfinished summary
* [Page 13: 2018-06-11  ](#id-section13). Thermocouple Calibrations
* [Page 14: 2018-06-12  ](#id-section14). Rhagoletis Data Revisions & upd.  to do list for projects
* [Page 15: 2018-06-12  ](#id-section15). Sorting out the coefficients of models for Rhagoletis
* [Page 16: 2018-06-13 ](#id-section16). ECB Setup Workflow & other tasks
* [Page 17: 2018-06-13 ](#id-section17). Notes and Data from yesterday's practice HS
* [Page 18: 2018-06-13 ](#id-section18). Practice Heat Shock at 37ºC
* [Page 19: 2018-06-14  ](#id-section19). Protists Project Notes
* [Page 20: 2018-06-14 ](#id-section20). Practice Heat Shock at 38ºC & ECB diet notes
* [Page 21: 2018-06-15 ](#id-section21).  Practice Heat Shock at 38.5ºC & tasks update
* [Page 22: 2018-06-18 ](#id-section22). Plan for this week
* [Page 23: 2018-06-19 ](#id-section23).  Fast Ramp Practice HS
* [Page 24: 2018-06-20 ](#id-section24). 37.5ºC Practice Static HS and Slow Ramp Practice HS
* [Page 25: 2018-06-21 ](#id-section25). Slow Ramp - redo, today's tasks
* [Page 26: 2018-06-22 ](#id-section26). Gilchrist Papers on Drosophila HS
* [Page 27: 2018-06-25  ](#id-section27). Cohort Setup for thermal assays
* [Page 28: 2018-06-26 ](#id-section28). DSPR Drosophila Flipping Schedule 
* [Page 29: 2018-06-27 ](#id-section29). Hardening Heat Shock Test & Protein Quant Test
* [Page 30: 2018-06-28 ](#id-section30). Protein Isolation II & Freezer box
* [Page 31: 2018-06-29 ](#id-section31). Flipping DSPR lines, Gioia + Zars 2009 & Jensen et al. 2010 papers, Test static for DSPR
* [Page 32: 2018-07-01 ](#id-section32). Cohort 1 Tests
* [Page 33: 2018-07-02 ](#id-section33). Hardening HS Test with DSPR Lines
* [Page 34: 2018-07-03 ](#id-section34). Cohort 2 Tests
* [Page 35: 2018-07-04 ](#id-section35). Survivorship Checks
* [Page 36: 2018-07-05 ](#id-section36).  Paper: Adaptation of Enzymes to Temperature (Somero 1968)
* [Page 37: 2018-07-06 -- 2018-07-10  ](#id-section37).  Thermal Assays Cohorts 1-5
* [Page 38: 2018-07-11 ](#id-section38). Plan for the rest of this week
* [Page 39: 2018-07-12 ](#id-section39). Thermal Assay Cohort 6 - Statistical approaches
* [Page 40: 2018-07-13 -- 2018-07-15 ](#id-section40). Thermal Assay Cohort 7-8
* [Page 41: 2018-07-13 ](#id-section41). YT Video Script draft 1
* [Page 42: 2018-07-15 ](#id-section42). YT video script draft 2
* [Page 43: 2018-07-16 -- 2018-07-17  ](#id-section43). Cohort 8-10
* [Page 44: 2018-07-18 ](#id-section44). Protist Project Upd
* [Page 45: 2018-07-18 ](#id-section45). Script draft 3
* [Page 46: 2018-07-19 ](#id-section46).  Protein Quantification
* [Page 47: 2018-07-20 ](#id-section47). Protein Homogenization Comparison
* [Page 48: 2018-07-23 ](#id-section48). Protein Unfolding
* [Page 49:  ](#id-section49).
* [Page 50:  ](#id-section50).
* [Page 51:  ](#id-section51).
* [Page 52:  ](#id-section52).
* [Page 53:  ](#id-section53).
* [Page 54:  ](#id-section54).
* [Page 55:  ](#id-section55).
* [Page 56:  ](#id-section56).
* [Page 57:  ](#id-section57).
* [Page 58:  ](#id-section58).
* [Page 59:  ](#id-section59).
* [Page 60:  ](#id-section60).
* [Page 61:  ](#id-section61).
* [Page 62:  ](#id-section62).
* [Page 63:  ](#id-section63).
* [Page 64:  ](#id-section64).
* [Page 65:  ](#id-section65).
* [Page 66:  ](#id-section66).
* [Page 67:  ](#id-section67).
* [Page 68:  ](#id-section68).
* [Page 69:  ](#id-section69).
* [Page 70:  ](#id-section70).
* [Page 71:  ](#id-section71).
* [Page 72:  ](#id-section72).
* [Page 73:  ](#id-section73).
* [Page 74:  ](#id-section74).
* [Page 75:  ](#id-section75).
* [Page 76:  ](#id-section76).
* [Page 77:  ](#id-section77).
* [Page 78:  ](#id-section78).
* [Page 79:  ](#id-section79).
* [Page 80:  ](#id-section80).
* [Page 81:  ](#id-section81).
* [Page 82:  ](#id-section82).
* [Page 83:  ](#id-section83).
* [Page 84:  ](#id-section84).
* [Page 85:  ](#id-section85).
* [Page 86:  ](#id-section86).
* [Page 87:  ](#id-section87).
* [Page 88:  ](#id-section88).
* [Page 89:  ](#id-section89).
* [Page 90:  ](#id-section90).
* [Page 91:  ](#id-section91).
* [Page 92:  ](#id-section92).
* [Page 93:  ](#id-section93).
* [Page 94:  ](#id-section94).
* [Page 95:  ](#id-section95).
* [Page 96:  ](#id-section96).
* [Page 97:  ](#id-section97).
* [Page 98:  ](#id-section98).
* [Page 99:  ](#id-section99).
* [Page 100:  ](#id-section100).
* [Page 101:  ](#id-section101).
* [Page 102:  ](#id-section102).
* [Page 103:  ](#id-section103).
* [Page 104:  ](#id-section104).
* [Page 105:  ](#id-section105).
* [Page 106:  ](#id-section106).
* [Page 107:  ](#id-section107).
* [Page 108:  ](#id-section108).
* [Page 109:  ](#id-section109).
* [Page 110:  ](#id-section110).
* [Page 111:  ](#id-section111).
* [Page 112:  ](#id-section112).
* [Page 113:  ](#id-section113).
* [Page 114:  ](#id-section114).
* [Page 115:  ](#id-section115).
* [Page 116:  ](#id-section116).
* [Page 117:  ](#id-section117).
* [Page 118:  ](#id-section118).
* [Page 119:  ](#id-section119).
* [Page 120:  ](#id-section120).
* [Page 121:  ](#id-section121).
* [Page 122:  ](#id-section122).
* [Page 123:  ](#id-section123).
* [Page 124:  ](#id-section124).
* [Page 125:  ](#id-section125).
* [Page 126:  ](#id-section126).
* [Page 127:  ](#id-section127).
* [Page 128:  ](#id-section128).
* [Page 129:  ](#id-section129).
* [Page 130:  ](#id-section130).
* [Page 131:  ](#id-section131).
* [Page 132:  ](#id-section132).
* [Page 133:  ](#id-section133).
* [Page 134:  ](#id-section134).
* [Page 135:  ](#id-section135).
* [Page 136:  ](#id-section136).
* [Page 137:  ](#id-section137).
* [Page 138:  ](#id-section138).
* [Page 139:  ](#id-section139).
* [Page 140:  ](#id-section140).
* [Page 141:  ](#id-section141).
* [Page 142:  ](#id-section142).
* [Page 143:  ](#id-section143).
* [Page 144:  ](#id-section144).
* [Page 145:  ](#id-section145).
* [Page 146:  ](#id-section146).
* [Page 147:  ](#id-section147).
* [Page 148:  ](#id-section148).
* [Page 149:  ](#id-section149).
* [Page 150:  ](#id-section150).

------

<div id='id-section1'/>

### Page 1: 2018-01-22.  First Day, tasks, and potential summer projects

Downloaded and learned to use:
* Typora
* Atom (for R markdown)
* GitHub

Shadowed Kylie for Rhagoletis rearing
* Transferring insects to Trikinetics setup
* Documenting in notebook and spreadsheet

Discussed with Andrew a list of tasks for me to complete:
1. European cornborer rearing (training with Qinwen) and writing up protocols (using Rmarkdown)
2. Contribute to the Rhagoletis project:
   * Shadow Kylie/Tatiana
   * Checking eclosions, deaths
   * Transferring entrainment samples to free run
   * Documentation in master spreadsheet and in notebook

3. Maintain personal notebook on GitHub
   * to do lists
   * read papers
   * experimental notes
   * seminar notes
   * etc.

Potential projects to discuss with Andrew and Dr. Hahn
* ECB: Timing of protein-protein interactions underlying seasonal changes in diapause
* Proteome stability project: What are the timing of events for proteostasis?
* Rhagoletis data mining: What are the consequences of diapause depth for overwinter survival?

Papers to read:
* Dopman, E. B., Bogdanowicz, S. M., & Harrison, R. G. (2004). Genetic Mapping of Sexual Isolation Between E and Z Pheromone Strains of the European Corn Borer (Ostrinia nubilalis). Retrieved from http://ase.tufts.edu/biology/labs/dopman/documents/pub2004GeneticMapping.pdf

* Dopman, E. B., Robbins, P. S., & Seaman, A. (2009). COMPONENTS OF REPRODUCTIVE ISOLATION BETWEEN NORTH AMERICAN PHEROMONE STRAINS OF THE EUROPEAN CORN BORER. Evolution, 64(4), 881–902. https://doi.org/10.1111/j.1558-5646.2009.00883.x

* Leuenberger, P., Ganscha, S., Kahraman, A., Cappelletti, V., Boersema, P. J., von Mering, C., … Picotti, P. (2017). Cell-wide analysis of protein thermal unfolding reveals determinants of thermostability. Science (New York, N.Y.), 355(6327), eaai7825. https://doi.org/10.1126/science.aai7825

* Levy, R. C., Kozak, G. M., Wadsworth, C. B., Coates, B. S., & Dopman, E. B. (2015). Explaining the sawtooth: latitudinal periodicity in a circadian gene correlates with shifts in generation number. Journal of Evolutionary Biology, 28(1), 40–53. https://doi.org/10.1111/jeb.12562

* Feder, J. L., Berlocher, S. H., Roethele, J. B., Dambroski, H., Smith, J. J., Perry, W. L., … Aluja, M. (n.d.). Allopatric genetic origins for sympatric host-plant shifts and race formation in Rhagoletis. Retrieved from http://www.pnas.org/content/pnas/100/18/10314.full.pdf

* Tauber, C. A., & Tauber, M. J. (1981). Insect seasonal cycles: Genetics and Evolution. Ann. Rev. EcoL Syst, 12, 281–308. Retrieved from https://www.annualreviews.org/doi/pdf/10.1146/annurev.es.12.110181.001433

* Tauber, E., Zordan, M., Sandrelli, F., Pegoraro, M., Osterwalder, N., Breda, C., … Costa, R. (2007). Natural Selection Favors a Newly Derived timeless Allele in Drosophila melanogaster. J. Exp. Biol. B Mol. Dev. Evol. Dev. Biol. J. Biol. Chem. BMC Biol. Mar. Biotechnol. Biol. Bull. Proc. Natl. Acad. Sci. U.S.A. Evol. Dev, 316, 1895–1898. https://doi.org/10.1126/science.1141560

* XIE, X., MICHEL, A. P., SCHWARZ, D., RULL, J., VELEZ, S., FORBES, A. A., … FEDER, J. L. (2008). Radiation and divergence in the Rhagoletis Pomonella species complex: inferences from DNA sequence data. Journal of Evolutionary Biology, 21(3), 900–913. https://doi.org/10.1111/j.1420-9101.2008.01507.x

* Xie, X., Rull, J., Michel, A. P., Velez, S., Forbes, A. A., Lobo, N. F., … Feder, J. L. (2007). HAWTHORN-INFESTING POPULATIONS OF RHAGOLETIS POMONELLA IN MEXICO AND SPECIATION MODE PLURALITY. Evolution, 61(5), 1091–1105. https://doi.org/10.1111/j.1558-5646.2007.00091.x

------

<div id='id-section2'/>

### Page 2: 2018-05-30 Rhagoletis Viability Selection Project Overview

Rhagoletis - Biological rhythms

Ways to Talk about the system:
* pest management
* target control
* evolutionarily based

 **Big Question for the project**
* Is there greater viability selection for flies that have high metabolic rate? (could be better question)
* What are the consequences of metabolic rate on adult life span?
* how does metabolic rate —> survival 

**Hypothesis:**
* Flies with lower metabolic rate have a longer life span under overwintering conditions.
  * Lower metabolic rate --> less energy used --> longer lifespan 
*  Investing in high metabolic rate results in low adult survival

**Prediction:**
* Flies reared under favorable conditions will show less of a negative correlation between adult lifespan and metabolic rate. 

**Calculating metabolic rate:**
CO2 output / Δpurge unit time (hr)  * unit mass
adult life span = death date - eclosion date

#### Workflow
**Library**
```{r}
#install.packages("DiagrammeR")
library(DiagrammeR)
```
**Overall Workflow**
```{r}
mermaid("
  graph TD
  A[Master Data Sheet] --calculate--> B[Metabolic Rates CO2 production per hr per mass]
  B --> T[Day 11]
  B --> R[Day 15]
  A -- calculate --> C[lifespan]
  T --> E[Scatter plot showing relationship between life span and MR]
  R --> E
  C --> E
        ")

```
![workflow1](https://github.com/HannahHChu/Notebooks/blob/master/Images/workflow1_rhagoletis.png)

**Detailed workflow for calculating metabolic rate**
How do we calculate metabolic rate?
Metabolic Rate = CO2 production/ hours
Mass Specific Metabolic Rate = CO2 production/hrs per mass

* What are cohorts?
	* Essentially handling replicates because we cannot sample all flies at once. For each cohort, there are two batches: (1) licor 7000 and (2) licor 6262
	* To process more samples, measurements were spread across 2 respirometers(licors).
	* Each licor has associated tape color indicated in tape column
	* When purged (allowing pupae to breathe in a syringe under standardized air without CO2), there was a start and end time for beginning and end of purge for each tape samples (licor batch). 

* What columns do we need from the master data sheet?
	* mass_day10: mass on day 10 (mgs)
	* purge_time_1: initial time for pupae to breathe in tube
	* resp_time_1: time of respirometry sampling
	* resp_day_11: respirometry on day 11

**Problem** with purge_time_1: only start and stop times except for apple cohort 1 
**Solution** create time sequence for each cohort from start to end

```{r}
mermaid("
  graph TD
  A[Master Data Sheet] --> B[Create time sequence for purge_time_1 in hrs minutes]
  B--subtract from 24 and add to resp_time_1 --> C[Total hrs]
  C --divide over resp_day11--> E[metabolic rate]
  C --multiply by mass and divide over resp_day11--> D[Mass specific metabolic rate]
")
```
![workflow2](https://github.com/HannahHChu/Notebooks/blob/master/Images/workflow2_rhagoletis.png)


**Session Info**
```{r}
sessionInfo()
```


**Today's Tasks**
- [ ] Update OS
- [x]  Update Google Calendar
- [x] Shadowed Kylie for Rhagoletis rearing
  - Transferring insects to Trikinetics setup
  - Documenting in notebook and spreadsheet
- [x] Read: Kostal, Vladimir - Eco-physiological phases of insect diapause

------

<div id='id-section3'/>

### Page 3: 2018-05-30 Kostal paper -  notes
**Vocabulary**
* dormancy - any state of suppressed development (developmental arrest), which is adaptive (ecologically or evolutionarily meaningful), and usually accompanied with metabolic suppression
* quiescence - immediate response (without central regulation) to a decline of any limiting environmental factor(s) below the physiological thresholds with immediate resumption of the processes if the factor(s) rise above them.
* diapause - a more profound, endogenously and centrally mediated interruption that routes the developmental programme away from direct morphogenesis into an alternative diapause programme of succession of physiological events
	* the start of diapause usually precedes the advent of adverse conditions and the end of diapause may not coincide with the end of adversity
* cryptobiosis -  an extreme depth of metabolic suppression --  ‘‘a peculiar state of biological organization when the organism shows no visible signs of life and when its metabolic activity becomes hardly measurable, or comes reversibly to a standstill"
* Induction phase - occurs during genotype specific sensitive periods when cues from the environment are perceived and transduced into switching the ontogenetic pathway from direct development to diapause when the token stimuli reach some critical level (the response may be modified by other environmental factors).
* Preparation phase - occurs where the phases of diapause induction and initiation are separated by a period of direct development during which the individual is covertly programmed for later expression of diapause (behavioral & physiological)
* diapause intensity - relative duration of developmental arrest (diapause) at a given moment and under given environmental conditions
*  Initiation phase - direct development (morphogenesis) ceases, usually followed by regulated metabolic suppression.
	*   Mobile diapause stages may continue accepting food, building of energy reserves and seeking suitable microhabitat. Physiological preparations for the period of adversity may take place and intensity of diapause may increase.
* Maintenance phase - endogenous developmental arrest persists while the environmental conditions are favourable for direct development. 
	* Specific token stimuli may help to maintain diapause (prevent its termination). Metabolic rate is relatively low and constant. Unknown physiological process(es) lead to more or less gradual decrease of diapause intensity and increase of sensitivity to diapause terminating conditions.
* Termination phase - specific changes in environmental conditions stimulate (accelerate or resume) the decrease of diapause intensity to its minimum level and thus synchronize individuals within a population. 
	* By the end of the termination phase, a physiological state is reached in which direct development may overtly resume (if the conditions are permissive) or covert potentiality for direct development is restored but not realized (if the conditions are not permissive).
* Post-diapause quiescence - exogenously imposed inhibition of development and metabolism, which follows the termination of diapause when conditions are not favourable for resumption of direct development.

**Important quotations**
1.  "two main difficulties hinder attempts to define useful terms: (a) missing overt markers for intrinsic changes and (b) attempts to arbitrarily subdivide those phenomena that essentially are continuous."
2.  The correct characterization and description of the particular phase of diapause ... critical for the interpretation of physiological and molecular data
3.  (1) pre-diapause (2) diapause and (3) post- diapause
4. " egg diapause in the silkworm Bombyx mori and the pupal diapause in the flesh fly Sarcophaga bullata,  show that diapause induction leads to specific alterations in gene transcription, neuroendocrine milieu and metabolic pathways"
	* the individual is destined for later entry into a developmental arrest
5. (1) initiation, (2) maintenance and (3) termination.
6. (unknown upstream factors - neurope- tides - hormones), or competency of target tissues (hormonal receptors - members of signalling cascades - cell-cycle regulators -, etc.), or both, ... the roles of other factors, such as heat shock proteins, have also received attention
7. Examples in the literature either confirm that the initiation phase has a relatively high temperature optimum (Johnsen et al., 1997), or show that relatively higher temperatures during this phase result in developing a more intense diapause of longer duration and with higher survival rate after its termination
8. Hodek (1983) suggested distin- guishing between ‘‘horotelic’’ (evolving at the standard rate) and ‘‘tachytelic’’ (evolving at a rate faster than the standard) ways of diapause termination.

**Thoughts & Questions**
1. two problems: (a) gap in knowledge in genes that correspond to phenotypic changes during diapause (b) attempting to use discrete variables to describe a continuous process
2. Author presents solutions to the presented problems: (a) correctly characterize and describe phases of diapause to interpret when presented with molecular data
3. Author divides development into three broad phases with smaller subphases
4. examples already demonstrate that diapause inductions alter gene transcription...not gene transcription altering "decision" to diapause?
5. sub phases of diapause...author claims 
6. Difficult for physiologists to identify the moment diapause starts - so look to molecular level which points to regulatory factors such as these --> this makes sense since hormones regulate physiology and behavior...no way to determine on a physiological level?
7. Wouldn't high temperatures start to increase mr again?  read  referenced paper for more understanding
8. Does  ‘‘horotelic’’  ‘‘tachytelic’’ termination depend on mr?


Reference: Kostal, V. (2006). Eco-physiological phases of insect diapause. Journal of Insect Physiology, 52, 113–127. https://doi.org/10.1016/j.jinsphys.2005.09.008

------

<div id='id-section4'/>

### Page 4: 2018-05-31 R coding & ECB rearing; Dopman paper - introduction notes

* dplyr package
	* calculated days 11 & 15 metabolic rate, mass specific metabolic rate, and eclosion
	* data sheet incomplete for adult death dates -- lifespan needs missing information

* ECB rearing - shadowed

**Dopman paper**
<u>Introduction</u>

* biological species concept (BSC) - speciation caused by reproductive isolating barriers that prevent gene flow
   * two methods to study:
   	* (a) comparative approach - understand how isolating barriers evolve by measuring  strength of reproductive isolation for several barriers across a large group of taxa that vary in divergence time
        	* used to estimate the sequential order of isolating-barrier evolution (i.e. which evolved first, second, etc.) or to identify biological traits associated with enhanced diversification
			* Problem: not so simple, many barriers contribute to speciation
   	(b) "case study" approach - evaluates many different forms of reproductive isolation that could prevent a small number of closely related populations from reproduction 
   		* how strength of isolation varies among all of the individual components

* Current debate on  role of one or a few strong barriers to gene flow vs many weaker barriers 
  * relates to the ease in which speciation evolves
  * if barrier strength  increases with time, a barrier cannot have played a major historical role during speciation unless its contemporary strength is large

* problem: relative contribution of the strength of individual components to total cumulative reproductive isolation
  * barrier may disproportionately affect gene flow if it occurs at early life stages, before other barriers have had their effect

* Quantified strength of reproductive isolation for:
  * seasonal temporal isolation
  * circadian temporal isolation
  * male behavioral isolation
  * female behavioral isolation, mechanical isolation
  * gametic isolation (oviposition and fertilization)
  * F1 hybrid inviability
  * behavioral hybrid (male) infertility
  * physiological hybrid infertility (oviposition and fertilization)
  * F2 and BC inviability

* Three goals:
	(i) to confirm past data on reproductive barriers and formally quantify isolation strength
	(ii) to evaluate new potential reproductive barriers
	(iii) to address the three outstanding issues in speciation research that case studies are well suited to confront

Reference: Dopman, E. B., Robbins, P. S., & Seaman, A. (2009). COMPONENTS OF REPRODUCTIVE ISOLATION BETWEEN NORTH AMERICAN PHEROMONE STRAINS OF THE EUROPEAN CORN BORER. Evolution, 64(4), 881–902. https://doi.org/10.1111/j.1558-5646.2009.00883.x

------

<div id='id-section5'/>

### Page 5: 2018-06-01 More R coding & new project plan
**To Do List**
- [X] Finish plotting predicted values
	-scatter plot with lifespan on y-axis and MR on x-axis
	-real data are points
	-overlay with predicted values
	-pred values displayed as line geom_line() - may need to specify data set in the function
	-explain the patterns
- [X]  Create an experimental plan for ECB system
	- Want to measure biological rhythms for each strain
		- sample size, conditions (FC vs diapause)
		- food for adults
		- replicates of cohorts (~5)
- [X]  Go over research plan
- [X]  Look into G Matrices
- [X]  Shadow Tatiana for evening check

------

<div id='id-section6'/>

### Page 6: 2018-06-03 G Matrix & ECB Exp. Plan
**G Matrix**
Background:
* Traits are determined by many factors: genetics and environmental
* Proportion of the phenotypic variation due to additive genetic effects for a single trait is described by the heritability (h2) - ranges from 0 to 1
* If the heritability is 1, then an offspring is exactly intermediate between the two parents
* If heritability is 0, the offspring would be no more similar to its own parents than to a random individual from the population
* Heritability describes how a trait will respond to selection
* Traits with more genetic variation respond more rapidly to natural or artificial selection, and the response to selection is given by the breeder’s equation: 
$$
R = h^2S
$$
What is the G-matrix and why is it important?
* Organisms have multiple traits and some are related at the level of the genome
* G-matrix describes genetic correlations: several traits may be affected by the same set of genes, and selection on one of the traits would result in changes in the others. 
*  G-matrix tells us how much genetic variation each trait has, as well as the extent to which the traits are genetically correlated with one another.
* For two traits, the G-matrix looks like this:
$$
\mathbf{G} = \left[\begin{array}
{rrr}
G_{11} & G_{12} \\
G_{12} & G_{22}
\end{array}\right]
$$
where $$ G_{11} $$ and $$ G_{22} $$ are the additive genetic variances for trait one and trait two and $$ G_{12} $$ is the additive genetic covariance (covariance is similar to the correlation, except correlations are standardized to take values between -1 and +1)

Multivariate breeder’s equation:
$$
\Delta {z} = G\beta
$$

Δz is a vector of changes in mean trait values, and β is a vector of selection gradients for the traits
[Genetics & G matrix](http://www.bio.tamu.edu/index.php/faculty/jones/lab/gmatrix/whatisg/)

**Experimental plan for ECB system**
Goal:  To measure biological rhythms for each strain (UZ & BE)
* Sample Size: cohorts of each strain in each condition (FC (a)  vs diapause (b))
	* 5 cohorts of 50 = total 500 individuals
	* 5 cohorts of 100 = total 1000 individuals
	* 5 cohorts of 150 (is this feasible?) = total 1500 individuals
	* 4 cohorts of 150 = total 1200 individuals 
* Conditions:
	* Favorable conditions: 16:8 L:D 24ºC or 26ºC (simulates day length for active reproductive months(May and June)) - keep in these conditions for ~60 days?
	* Diapause conditions: 12:12 L:D 23ºC (46 days according to previous protocol)

1) Cohort A & B in FC for 60 days (or however long it takes to become 5th instar/prepupation)
2) Cohort A keep in FC for 46 days & Cohort B move to diapause for 46 days in Trikinetics
3) Remove both cohorts back to FC

Reference: Dopman, E. B., Robbins, P. S., & Seaman, A. (2009). COMPONENTS OF REPRODUCTIVE ISOLATION BETWEEN NORTH AMERICAN PHEROMONE STRAINS OF THE EUROPEAN CORN BORER. Evolution, 64(4), 881–902. https://doi.org/10.1111/j.1558-5646.2009.00883.x

------

<div id='id-section7'/>

### Page 7: 2018-06-04 Survival Data Analysis
#### Workflow for Merging Pupal Death Data with Lifespan Data
```{r}
mermaid("
  graph TD
  A[Master Data Sheet] --filter out the non-eclosers--> B[Subset of ind. who haven't eclosed]
  A -- adult death date - eclosion date --> E[Lifespan Data]
  C[Pupal Deaths Data Sheet]
  B --merge based on host, cohort_day, well_id, uniqueID --> D[Merged Data Sheets]
  C --merge based on host, cohort_day, well_id, uniqueID --> D
  D --merge--> F[Final Data Sheet]
  E --merge--> F[Final Data Sheet]
  F --analyses using R --> G[negative binomial regression]
  F --analyses using R package survival --> H[Survival analysis]
  F --analyses using R--> I[COX regression]
")
```
![workflowpdd](https://github.com/HannahHChu/Notebooks/blob/master/Images/workflow3pdd.png)

#### Survival Data Analysis
[Regression Models for Survival Data](https://www.stat.ubc.ca/~rollin/teach/643w04/lec/node63.html)
* Survival analysis,  failure time analysis, or time-to-event analysis methods used to:
	*  examine the time to occurrence of an event (i.e. death, occurrence of morbidity, remission of disease, failure of a component, etc.)
* Response (or dependent) variable, *y* = time to the occurrence of the event relative to initial time
* If follow-up ends before the event occurence,  the subject's survival time has been *censored*
	* We don't know the exact value of *y*, only that *y > t* (the actual duration of follow-up)
	* With censored survival data, the response consists of two parts - 
		* duration of follow-up (continuous)
		* occurrence/non-occurence of the event (binary)
	* The last variable is sometimes called the censoring variable because it indicates whether censoring occured for the subject

[Survival Analysis Basics in R](http://www.sthda.com/english/wiki/survival-analysis-basics)
[Cox Regression in R](http://www.sthda.com/english/wiki/cox-proportional-hazards-model)




**To Do List**
- [X] Rhagoletis viability 
	- include pupae that did not emerge as adults in data (adult lifespan 0)
	- Fill out missing info in freezer datasheet
	- Create a dataset of pupae that have not eclosed (NB #002 pg 78-80)
		- Host, cohort_day, well ID, ind ID, life spa
		- Double check pupae have not eclosed 
		- Link that data into master spreadsheet using R
- [X] Update storage sheet
- [ ] Read Plata & Vitkup (2017) and Leuenberger, et. al. (2017)

------

<div id='id-section8'/>

### Page 8: 2018-06-05 Proteome Stability Programming

**To Do List**
- [ ] Rhagoletis Viability Selection
	- Fix R Chunks
	- Knit (Update OS)
	- Describe in terms of biology
	- Remove mass for figures for neg. binomial regression
	- Make figures prettier (add units [mL CO2/hr], larger font)
- [ ] Proteome Stability - set up 4 programs on Wintherm Plus (with initial 5 min incubation pd):
	- Ctmax slow ramp (0.1ºC/min from 25ºC-40ºC)
	- Ctmax fast ramp (1ºC/min from 25ºC-40ºC)
	- Static (2 hrs at 39ºC)
	- Hardening (36ºC for 1 hr, 25ºC for 1 hr, 39ºC for 1 hr)
	- Ask Tatiana about drosophila rearing temp

Reference: Bettencourt, Brian R., et al. "Inducible and constitutive heat shock gene expression responds to modification of Hsp70 copy number in Drosophila melanogaster but does not compensate for loss of thermotolerance in Hsp70 null flies." BMC biology 6.1 (2008): 5.

------

<div id='id-section9'/>

### Page 9: 2018-06-06 Proteome Stability Project To Do List
**To Do List**
- [X] Create rack for water bath (14 tubes)
![waterbathrack](https://github.com/HannahHChu/Notebooks/blob/master/Images/water_bath_tubes4.JPG)
- [ ] Write out and add protocol to HahnLab protocols (Refer to Andrew's website) including:
	- brand
	- images
	- catalog #
	- vendor ID
- [ ] Develop ramping programming based on the apparatus inside the water bath
- [X] Find a way to keep the thermocoupler stable in the water bath
- [X] Plot the data from ramping tests
- [X] Evening check for free runs

#### Ramping Protocol Testing
**Static**
<u>Test 1</u>
![test1c](https://github.com/HannahHChu/Notebooks/blob/master/Images/test1chart.png)
![test1](https://github.com/HannahHChu/Notebooks/blob/master/Images/static_protocol_t1.png)

| Time (min) | Temp (ºC) |
| -- 		 | --------- |
| 1  		 | 25.1  |
| 2  		 | 25.1  |
| 3  		 | 25.1  |
| 4  		 | 25.1  |
| 5  		 | 25.1  |
| 6  		 | 25.7  |
| 7  		 | 26.6  |
| 8  		 | 27.7  |
| 9  		 | 28.7  |
| 10		 | 30.0  |
| 11 		 | 32.0  |

<u>Test 2</u>
![test2c](https://github.com/HannahHChu/Notebooks/blob/master/Images/test2chart.png)
![test2](https://github.com/HannahHChu/Notebooks/blob/master/Images/static_protocol_t2.png)

| Time (min) | Temp (ºC) |
| -- 		 | --------- |
| 1  		 | 25.1  |
| 2  		 | 25.1  |
| 3  		 | 25.1  |
| 4  		 | 25.1  |
| 5  		 | 25.1  |
| 6  		 | 25.9  |
| 7  		 | 27.3  |
| 8  		 | 29.2  |
| 9  		 | 31.6  |
| 10		 | 33.8  |
| 11 		 | 35.7  |
| 12 		 | 37.7  |
| 13 		 | 39.9  |

<u>Test 3</u>
![test3c](https://github.com/HannahHChu/Notebooks/blob/master/Images/test3chart.png)
![test3](https://github.com/HannahHChu/Notebooks/blob/master/Images/static_protocol_t3.png)

| Time (min) | Temp (ºC) |
| -- 		 | --------- |
| 1  		 | 25.1  |
| 2  		 | 25.1  |
| 3  		 | 25.1  |
| 4  		 | 25.1  |
| 5  		 | 25.1  |
| 6  		 | 25.9  |
| 7  		 | 27.3  |
| 8  		 | 29.2  |
| 9  		 | 31.6  |
| 10		 | 33.8  |
| 11 		 | 35.7  |
| 12 		 | 37.7  |
| 13 		 | 39.9  |

------

<div id='id-section10'/>

### Page 10: 2018-06-07 Water Bath Tests
- [X] Tested ramping for slow and fast - refer to pages 10-15 in NB#001.
Problem with fast ramp at 32ºC, the water bath either speeds up too much or slows down too much.

- [X] Nightly check for free run and eclosion

------

<div id='id-section11'/>

### Page 11: 2018-06-08 Water Bath Tests II
- [X] Tested ramping for fast - refer to page 26 in NB#001.

| Time (min) | Temp (ºC) |
| -- 		 | --------- |
| 1  		 | 25.1  |
| 2  		 | 25.1  |
| 3  		 | 25.1  |
| 4  		 | 25.1  |
| 5  		 | 25.1  |
| 6  		 | 25.6  |
| 7  		 | 26.6  |
| 8  		 | 27.7  |
| 9  		 | 28.6  |
| 10		 | 29.6  |
| 11 		 | 30.7  |
| 12 		 | 31.7  |
| 13 		 | 32.5  |
| 14 		 | 33.5  |
| 15 		 | 34.5  |
| 16 		 | 35.5  |
| 17 		 | 36.5  |
| 18 		 | 37.5  |
| 18 		 | 38.5  |
| 20 		 | 39.5  |
| 21 		 | 40.0  |

- [X] Nightly check for free run and eclosion

------

<div id='id-section12'/>

### Page 12: 2018-06-10 Rhagoletis Viability Selection Analysis -- unfinished summary

**ANOVA**
ANOVA  tests for differences between two or more means
* The null hypothesis would be that the two means are equal
* Significance would indicate that two means are not equal

<u>One Way ANOVA</u> compares two means from two independent groups, in this case eclosion date and host (specific to SO and RT)

<u>Two Way ANOVA</u> compares means of two independent variables affecting one dependent variable, in this case the effect of Host and treatment (interacting) on eclosion

Both the one way and two way ANOVA tests showed significance between the two means (meaning they are not equal). The two way ANOVA suggests that there is a difference in number of days to eclosion based on treatment type - favorable conditions vs. simulated overwintering. Therefore, a one way ANOVA was done for each treatment to test whether there is any significance between host type and days to eclosion. The one way ANOVA showed significance for both treatments; however, the simulated overwintering samples had a lower p value, which indicates higher signigicance.

**Negative Binomial Regression**
Samples were split into two treatments, RT (favorable conditions) and SO (simulated overwintering). The dependent variable was lifespan, while the independent variables were metabolic rate in relation to host and mass. For this model, a negative coefficient of the log value would indicate a shorter lifespan since it produces a log-linear model in which a positive coefficient would indicate a (larger x and y value)?.

Based on the RT treatment, only mass showed significance which suggests an individual with a larger mass would have a longer lifespan (positive coefficient). Analysis of the SO treatment showed significance in host type, metabolic rate, and mass, in which a lower metabolic rate would suggest longer lifespan (neg. coefficient), and larger mass would suggest longer lifespan(pos. coefficient). Hawthorne individuals also had an overall lower lifespan compared to apple. 

Negative binomial regression was shown to be a good model by comparing its log likelihood to a poisson distribution. Values close to 0 (which estimates the dispersion parameter) strongly suggests the NBR model is more appropriate than the Poisson model for both treatments.

**Cox Regression**
Samples were split into two treatments, RT (favorable conditions) and SO (simulated overwintering). The dependent variable was lifespan, while the independent variables were metabolic rate in relation to host and mass. For this model, the coefficient relates x variables (metabolic rate, host, and mass) to likelihood of death. The dependent variable is lifespan again.

The RT treatment only showed significance for mass (negative coefficient), in which a lower mass suggests a lower likelihood of death (i.e. longer lifespan). ? doesnt agree with NBR

The SO treatment showed significance for metabolic rate, host, and mass. The positive coefficient for the hawthorne hosts and metabolic rate indicate higher likelihood of death if the individual is a host to hawthorne and/or the individual has a high metabolic rate. This agrees with the NBR model. However, the negative coefficient for mass indicates a lower likelyhood of death with smaller masses which disagrees with the NBR model. 

**Kaplan-Meier Survival Estimate**
The Kaplan-Meier Survival Estimate predicts the probability of survival from a specific time, so for this data, the estimates predict the probability of host survival at each time point in their lifespan. According to the figure, Hawthorne hosts have an overall lower probability of survival over apple hosts, which agrees with the previous analyses. This estimate is best used for categorical values, so MR and mass (which were all unique to the individul) were not good estimators and not used. 

**Conclusion**
Evolutionarily speaking, linking lower metabolic rate to longer lifespan makes sense since an organism who reserves its energy would take longer to develop, thus longer to die. According to the evolution theory described in Olshansky & Rattan (2009), higher metabolic rates are liked to species with "high extrinsic mortality" (e.g. targeted prey) since they must quickly develop and reproduce in order to pass down their genes. Lower metabolic rates were linked to individuals with lower extrinsic mortalities and could take their time to gradually develop into larger sizes and live longer. 

------

<div id='id-section13'/>

### Page 13: 2018-06-11 Thermocouple Calibrations
**To Do List**
- [ ] Write protocol for proteome stability project
- [X] Night free run check
- [X] Thermometer calibrations

##### ExTech Thermocoupler Easyview 15
[Manual](http://www.extech.com/resources/EA15_UM.pdf)
[Website](http://www.extech.com/display/?id=14443)
	Catalog No. EA15
![img](https://github.com/HannahHChu/Notebooks/blob/master/Images/IMG_8001.jpg)
```{r}
settemp<- c(0, 25, 30, 35, 40, 45, 50)
T1 <- c(0.7, 24.2, 29.1, 34.1, 39.2, 44.2, 49.1)
T2 <- c(1, 23.9, 28.7, 33.8, 38.8, 43.9, 48.7)
wide <- c(0, 25.05, 30, 35, 40, 45, 50)
narrow <- c(0.2, 25.1, 30, 35, 40, 45, 50)

fit<-lm(narrow~T1)
summary(fit)
plot(T1, narrow, main = "ExTech")
abline(fit)

fit2<-lm(narrow~T2)
summary(fit2)
plot(T2, narrow, main = "ExTech")
abline(fit2)
```

##### Omega Thermocoupler HH506RA
[Manual](https://www.omega.com/manuals/manualpdf/M4079.pdf)
[Website](https://www.omega.com/pptst/HH506A_HH506RA.html)
	Model No. HH506RA
![img](https://github.com/HannahHChu/Notebooks/blob/master/Images/IMG_8002.jpg)
```{r}
settemp<- c(25, 30, 35, 40, 45, 50)
T1o <- c(25, 32.7, 40.3, 47.9, 55.4, 64.6)
T2o <- c(25, 30, 35, 40.1, 45.2, 51.5)
wideo <- c(25.05, 30, 35, 40, 45, 50)
narrowo <- c(25.1, 30, 35, 40, 45, 50)

fito<-lm(narrow~T1)
summary(fito)
plot(T1o, narrowo, main = "Omega")
abline(0,1)

fit2o<-lm(narrow~T2)
summary(fit2o)
plot(T2o, narrowo, main = "Omega")
abline(0,1)
```

------

<div id='id-section14'/>

### Page 14: 2018-06-12 Rhagoletis Data Revisions & upd.  to do list for projects
- [X] Free Run Check
- [X] Work on Rmd file based on Andrew's comments:
	- Comment out long stretches of data code ('glimpse') √
	- Space after hashtags √
	- Include conclusion of the analysis in the Rmd file too. 
	- Summarise statistical analysis under the r chunk (ie We tested the effect of x,y on z. Or we determined the hazard rate of xy on z) √
	- Annotate filtering √

**notes about those conclusions**: 
"Evolutionarily speaking, linking lower metabolic rate to longer lifespan makes sense since an organism who reserves its energy would take longer to develop, thus longer to die". 

* This is interesting, but what is the cost of low vs high metabolic rates in terms of seasonal timing? 
* What is costly between the treatments that relate to the natural setting? I think you can expand on some thoughts in this area. 

"According to the evolution theory described in Olshansky & Rattan (2009), higher metabolic rates are liked to species with "high extrinsic mortality" (e.g. targeted prey) since they must quickly develop and reproduce in order to pass down their genes." 

* This statement somewhat contrasts the previous. Is it that lifespan is a function of metabolic rate? Or does selection weed out ones that express metabolic rates which don't match the environment optimally?
* What is the extrinsic mortality factor in this case? 

**To Do List**:
1. ECB project: we have an experimental approach outlined (SURF nb#001, page 32)
	* Basically, we want to set up 5 cohorts for each strain (UZ and BE). 
		* For each cohort, measure larval post diapause termination( transition from larvae to pupae) under favorable and unfavorable conditions. 
		* Then take the adults and measure their activity in the trikinetics set up. In the trikinetics set up, have sugar water on 1 side and water on the other (double check)
	* learn rearing protocol and add upon it in the existing protocol (check readme for formatting and structure)
	* create github repository and populate with documents, data, results
	* document experimental workflow in readme (also add a little background info on yourself and the system)
	* log the data in an excel spreadsheet + notebook
	* individual, strain, cohort, cohort date, pupation date, eclosion date, death date, trik enter date, trik exit date, trik position, trik monitor, (have free run version too) , sex

2. DSPR proteome stability project: Flies arrived today. We obtained 13 strains (SURF nb#001, page 34). Tested static today at 39C.
	* let flies sit for 2 weeks. Coordinate a rearing schedule
	* document activities in notebook
	* Try static HS at 37 C
	* Try other heating protocols and summarise the data in a R markdown doc. Can try cold tolerance experiments but need to change water bath.
	* Write up protocol in hahn lab protocol website github repo. Include info on vendor, pictures, and links for all materials.
	* when starting experiment; take adults from 5-7 days (they are fully mated)
	* check the line that may be missing (http://wfitch.bio.uci.edu/~dspr/index.html)

3. Viability selection coding proj:
	* Fixing pdf output to make it more presentable; send to ANBE by Wednesday √
	* Create a direction of effect table to make sense of the data. √
	* Show Dr. Hahn Friday. √

4. Calibration water bath
	* try calibration from 35-40C range. √
	* write up technical report in Rmarkdown √
	* clean water bath, it is cloudy. Dr. Hahn suggests to put some ethanol in it. (also leave it open so it doesn't get as cloudy) √
	* make a new rack, have tubes that can be inserted into the styrofoam for better handling √

------

<div id='id-section15'/>

### Page 15: 2018-06-12 Sorting out the coefficients of models for Rhagoletis

| Condition | Variable | Statistical Model | Coefficient | Meaning |
| -- 		 | ------- | --- | ---- | --- |
| RT  		 |   Mass  | NBR | pos. | larger mass, longer lifespan |
| RT  		 |   Mass  | Cox | neg. | larger mass, less likely to die |
| SO  		 |   Mass  | NBR | pos. | larger mass, longer lifespan |
| SO  		 |   Mass  | Cox | neg. | larger mass, less likely to die |
| SO  		 |    MR   | NBR | neg. | lower MR, longer lifespan |
| SO  		 |    MR   | Cox | pos. | higher MR, more likely to die |
| SO  		 |   Host  | NBR | neg. | Haw flies have overall shorter lifespan |
| SO  		 |   Host  | Cox | pos. | Haw flies are more likely to die |

------

<div id='id-section16'/>

### Page 16: 2018-06-13 ECB Setup Workflow & other tasks

[See workflow for ECB](https://github.com/HannahHChu/ECB/blob/master/Documents/2018-06-12_ECB_workflow.Rmd)

[See complete calibration summary](https://github.com/HannahHChu/Notebooks/blob/master/Proteostasis/Thermocoupler_calibration.pdf)

**Thermometer Omega T2 Narrow Calibration check from 34-40ºC**

| Set Temp (ºC) | Omega T2 Reading | Mercury Thermometer Reading |
| -- 		 	 | ------- | --- |
| 34  			 |   34  | 34 |
| 34.5  		 |   34.5  | 34.5 |
| 35  			 |   35  | 35 |
| 35.5  		 |   35.5  | 35.5 |
| 36	  		 |    36   | 36 |
| 36.5  		 |    36.6   | 36.5 |
| 37	  		 |   37  | 37 |
| 37.5  		 |   37.5  | 37.5 |
| 38	  		 |   38.1  | 38 |
| 38.5  		 |   38.6  | 38.5 |
| 39	  		 |   39.1  | 39 |
| 39.5  		 |   39.6  | 39.5 |
| 40	  		 |   40.1  | 40 |

------

<div id='id-section17'/>

### Page 17:2018-06-13 Notes for DSPR and Data from yesterday's practice HS

[DSPR Website](http://wfitch.bio.uci.edu/~dspr/index.html)
* Recieved 13 lines on 2018-06-12
*  Empty vials contain eggs and larvae
*  Make sure cotton cover is tight to prevent flies from escaping
*  Vials were labeled 11JUN2018 -- on Friday (2018-06-15), transfer those adults into a bottle
*  How to tell Male and Female apart:
	*  Males: front legs have "sex cones" for mounting, abdomen is narrower with a black tip
	*  Females have thicker abdomens that are all white (full with eggs)
*  Keep the old vial (vial with adults that were flipped into new vials) and wait 1 week for new generation of adults
*  Keep vials fresh, 21 max days for vial age
	*  This is to prevent mites: 2 miticides are sodium benzoate & probenoic acid

**What is DSPR (Drosophila Synthetic Population Resource)?**

* Drosophila melanogaster
	* 1700 recombinant inbred lines (RILs)
		* originates from 2 highly recombined synthetic populations
			* derived from set of 8 inbred samples
			* Share the A8 line
				* crossed in a "round robin" mating scheme
	* Genotyped ~10K SNPs
	* makes it easier to link genotype to phenotype since the entire genome only needs to be sequenced for the RIL and then look for SNPs for phenotypic differences. 

**Stock flies we are allowed to use**

H1, H2, C1, C2

**DSPR Lines Recieved** (Missing B5 line)

Reference: SURF NB #001 pg 34

| #   | Name | No. of Vials | Date 		|
| --- | ---- | ---------  | ------|
| 1  | AB8  |  3  	 	 |  2018-06-11	|
| 2  | A5 	|  2  		 |  2018-06-11  |
| 3  | A7   |  2  		 |  2018-06-11 	|
| 4  | A2 	|  2   	 	 | 2018-06-11	|
| 5  | B6   |  2    	 |  2018-06-11 	|
| 6  | B7 	|  2  		 |  2018-06-11 	|
| 7  | A4   |  2  		 |  2018-06-11 	|
| 8  | A6 	|  2  		 |  2018-06-11 	|
| 9  | A3   |  2  	 	 |  2018-06-11 	|
| 10 | B2 	|  2  		 |  2018-06-11 	|
| 11 | B3   |  2  		 |  2018-06-11 	|
| 12 | A1 	|  2  		 |  2018-06-11 	|
| 13 | B4   |  2  		 |  2018-06-11 	|

**Practice Heat Shock**

Protocol for Static at 39ºC

Lines used: C1, C2, H1, H2 (2018-05-21)

Knockdown: on back, inactivity for 5-10 sec

| Position   | Line | Time (M:S) | Notes 		|
| --- | ---- | ---------  | ------|
| 1  | H1  	|  09:20  	 	 |  	|
| 2  | C2 	|    		 |  Looked dead immediately  |
| 3  | H2   |   10:00		 |  Stopped moving for a little	|
| 4  | C1 	|  11:00   	 	 |  |
| 5  | C2   |  09:06    	 |  	|
| 6  | H1 	|  14:10  		 |  Stopped moving for a little	|
| 7  | H2   |  11:40  		 |  	|
| 8  | H1 	|  14:31  		 |  	|
| 9  | C2   |  10:30  	 	 |  	|
| 10 | C1 	|  09:40  		 |   	|
| 11 | H2   |  12:00  		 |  	|
| 12 | H2 	|  11:50  		 |  	|
| 13 | C2   |    		 	 |  Looked dead immediately 	|
| 14 | H1   |  11:30  		 |  	|

------

<div id='id-section18'/>

### Page 18: 2018-06-13 Practice Heat Shock at 37ºC

Protocol for Static at 37ºC

Lines used: C1, C2, H1, H2 (2018-05-28)

Knockdown: on back, inactivity for 5-10 sec

| Position   | Line | Time (M:S) | Notes 		|
| --- | ---- | ---------  | ------|
| 1  | C2  	|  34:54  	 	 | Male, fell over at 29:13	|
| 2  | C1 	|  87:22		 | Male, twitched for a long time   |
| 3  | H2   |  95:41		 |  Female	|
| 4  | H2 	|  94:19   	 	 | Female 	|
| 5  | C1   |  06:03    	 |  Female	|
| 6  | C1 	|  92:51  		 |  Male	|
| 7  | H1   |  79:30  		 |  Female	|
| 8  | C2 	|  69:38  		 |  Male	|
| 9  | H1   |  63:40  	 	 |  Female	|
| 10 | H2 	|  85:45  		 |  Female 	|
| 11 | C1   |  91:20  		 |  Female	|
| 12 | H1 	|  59:07  		 |  Female	|
| 13 | C2   |  98:24  	 	 |  Female 	|
| 14 | H1   |  98:38  		 |  Female	|

* Try 38ºC next time for static

------

<div id='id-section19'/>

### Page 19: 2018-06-14 Protists Project Notes

* Reconstruct phylogenies (gene level & concatonated genes)
	* Estimate the evolutionary substitution rate model
	* Cipres: jModelTest2 on XSEDE (2.1.6)   - Statistical selection of best-fit models of nucleotide substitution, run on XSEDE
		* Specify parameters to reconstruct phylogenies
* Build Trees with MLE (GARLI) or Bayesian (Mr. Bayes)
* Concatonated genes - talk to Andrew
* For R: read.tree function (newick file)

------

<div id='id-section20'/>

### Page 20: 2018-06-14 Protocol for Static at 38ºC & ECB diet notes

*Changed setup: using plastic test tube rack now*
*Not using the screw caps anymore, just use cotton and a 1x1 inch parafilm tightly wrapped around the opening*

Protocol for Static at 38ºC

Lines used: C1, C2, H1, H2 (2018-05-28)

Knockdown: on back, inactivity for 5-10 sec

| Position   | Line | Time (M:S) | Notes 		|
| --- | ---- | ---------  | ------|
| 1  | H2  	|  39:44  	 	 | Male	|
| 2  | C1 	|  58:32		 | Female  |
| 3  | C2   |  41:07		 |  Male	|
| 4  | H2 	|  60:02   	 	 | Female, down for 15 min, and got back up |
| 5  | H1   |  27:02    	 |  Female	|
| 6  | C1 	|  61:50  		 | Female	|
| 7  | H1   |  28:21  		 |  Female	|
| 8  | C2 	|  44:10  		 |  Male	|
| 9  | H2   |  47:07  	 	 |  Male	|
| 10 | C1 	|  29:38  		 |  Male 	|
| 11 | H1   |  32:30  		 |  Male	|
| 12 | H2 	|  48:15  		 |  Male	|
| 13 | C1   |  37:36  	 	 |  Male 	|
| 14 | C2   |  21:21  		 |  Male	|

**Future plans for HS Practice** 

* Try 38.5ºC and maybe 37.5ºC
* Slow ramp
* Plot KD temp vs KD time
![HS practice](https://github.com/HannahHChu/Notebooks/blob/master/Images/IMG_8034.JPG)

**ECB Adult Diet** by DA Andow & TJ Stodola - University of Minnesota

[Link](https://conservancy.umn.edu/bitstream/handle/11299/174415/ECB%20Manual%20Final.pdf?sequence=4&isAllowed=y)

* 10g Agar
* 710 mL dH2O
* 280g Sucrose

1. Heat agar and water to 90ºC
2. Add 280g sucrose
3. Stir and cool to 54ºC
4. Pour into mold

*Makes 1L*

------

<div id='id-section21'/>

### Page 21: 2018-06-15 Practice HS at 38.5ºC & tasks update

Protocol for Static at 38.5ºC

Lines used: C1, C2, H1, H2 (2018-06-05)

Knockdown: on back, inactivity for 5-10 sec

| Position   | Line | Time (M:S) | Sex 		|
| --- | ---- | ---------  | ------|
| 1  | C2  	|  24:33 	 	 | Female	|
| 2  | H1 	|  11:36		 | Female  |
| 3  | C1   |  06:27		 | Female	|
| 4  | H2 	|  30:36   	 	 | Female|
| 5  | H1   |  16:32    	 |  Female	|
| 6  | C1 	|  11:05  		 | Male	|
| 7  | C2   |  06:47  		 |  Female	|
| 8  | H2 	|  28:16  		 |  Male	|
| 9  | C1   |  15:22  	 	 |  Male	|
| 10 | C2 	|  25:19  		 |  Male 	|
| 11 | H1   |  21:26  		 |  Male	|
| 12 | C2 	|  07:29  		 | Female	|
| 13 | H1   |  17:33  	 	 |  Female 	|
| 14 | H2   |  08:48  		 | Female	|

**Tasks already completed**
- [X] Rhagoletis code 
- [X] Calibrated thermocouplers
- [X] Typed up thermocoupler calibration protocol and uploaded to Hahn lab protocols
- [X] Set up ECB UZ cage
- [X] Created ECB repository with master data sheet & workflow
- [X] Practice Heat Shocks at 37ºC, 38ºC, 38.5ºC, and 39ºC
- [X] Looked up ECB adult diet

**To complete**
- [ ] Slow ramp practice HS
- [x] Figure out new fast ramp protocol
- [ ] Finish writing protocol for thermal assays
- [ ] Set up ECB rearing
- [X] Create figures in R with static HS data
- [ ] Organize drosophila protocols

------

<div id='id-section22'/>

### Page 22: 2018-06-18 Plan for this week 

<u>Monday</u>
* Check up on ECB √
* Fast Ramp Correction √
* Update thermal assay protocol √
* Lab meeting (3:30) √

<u>Tuesday</u>
* Check up on ECB √
* Fast Ramp Practice HS √
* Rhagoletis meeting (12:00) √

<u>Wednesday</u>
* 37.5ºC Static HS√
* Slow Ramp Practice HS√

<u>Thursday</u>
* Work on plots for KD time√
* Work on protocols√

<u>Friday</u>
* Empty water bath & clean√
* Drosophila rearing w/ Tatiana 
* Work on plots for KD time√


**Test Fast Ramp (New) for calibrated Thermocoupler**

| Temp (ºC) | Time (min) | Tol (ºC) | Pump |
| -- 		 | --------- |  ------ | ------- |
| 25  		 |   5		 | 0 | 2 |
| 26  		 | 1  | 0 | 2 |
| 27  		 | 1  | 0 | 2 |
| 28  		 | 1  | 0 | 2 |
| 29  		 | 1  | 0 | 2 |
| 30  		 | 1  | 0 | 2 |
| 31  		 | 1  | 0 | 2 |
| 32  		 | 1  | 0 | 2 |
| 33  		 | 1  | 0 | 2 |
| 34		 | 1  | 0 | 2 |
| 35 		 | 1  | 0 | 2 |
| 36.5 		 | 1  | 0 | 2 |
| 37.5 		 | 1  | 0 | 2 |
| 38.5 		 | 1  | 0 | 2 |
| 40 		 | 60 | 0 | 2 |


| Time (min) | Temp (ºC) |
| -- 		 | --------- |
| 1  		 | 24.9  |
| 2  		 | 24.9  |
| 3  		 | 24.9  |
| 4  		 | 24.9  |
| 5  		 | 24.9  |
| 6  		 | 25.1  |
| 7  		 | 25.9  |
| 8  		 | 26.9  |
| 9  		 | 28.0  |
| 10		 | 28.9  |
| 11 		 | 29.9  |
| 12 		 | 31.1  |
| 13 		 | 32.2  |
| 14 		 | 33.2  |
| 15 		 | 34.1  |
| 16 		 | 35.0  |
| 17 		 | 35.9  |
| 18 		 | 37.0  |
| 18 		 | 38.0  |
| 20 		 | 38.9  |
| 21 		 | 40.0  |

------

<div id='id-section23'/>

### Page 23: 2018-06-19 Fast Ramp Practice HS

- [X] Fast ramp practice HS
- [X] Shadowed Tatiana for drosophila rearing 
- [X] Rhagoletis meeting 

**Fast Ramp Practice HS**
Protocol: refer to [pg 22](#id-section22)

Lines: C1, C2, H1, H2 (2018-06-05)

*Note: Used aspirator to transfer flies, not CO2*

| Position   | Line | Time (M:S) | Temperature (ºC)|  Sex 		|
| --- | ---- | ---------  | ------ | -------------- |
| 1  | H2  	|  39:11 	 	 | 39.1 |  Male	|
| 2  | H1 	|  39:22		 | 39.1 |  Male  |
|  3  | H2   |  48:32		 | 39.3 |  Male	|
| 4  | C2 	|  32:02   	 	 | 38.9 |  Female|
| 5  | C1   |  31:30    	 | 38.9 |  Male	|
| 6  | H1 	|  31:42  		 | 38.9 |  Male	|
| 7  | C1   |  36:55  		 | 39.0 |  Male	|
|  8  | H2 	|  20:37  		 | 38.5 |  Male	|
|  9  | C2   |  37:34  	 	 | 39.0 | Male	|
|  10 | H1 	|  46:29  		 | 39.3 | Female 	|
|  11 | C1   |  34:02  		 | 38.9 | Male	|
|  12 | C2 	|  41:50  		 | 39.1 | Female	|
|  13 | H1   |  40:37  	 	 | 39.1 | Female 	|
|  14 | C1   |  38:48  		 | 39.1 |Female	|

------

<div id='id-section24'/>

### Page 24: 2018-06-20 37.5ºC Practice Static HS and Slow Ramp Practice HS

Protocol: 37.5ºC

Lines: C1, C2, H1, H2 (2018-06-05)

* Note: Used aspirator to transfer flies, not CO2

| Position   | Line | Time (M:S) |  Sex |
| --- | ---- | --------- | -------------- |
| 1  | H1  	|  33:55 	 |  Male	|
| 2  | C1 	|  54:47	 |  Female  |
| 3  | H2   |  71:48	 |  Male	|
| 4  | C1 	|  78:52   	 |  Female	|
| 5  | C2   |  80:24     |  Female	|
| 6  | C1 	|  74:30  	 |  Male	|
| 7  | H2   |  32:24  	 |  Female	|
| 8  | C2 	|  44:10  	 |  Male	|
| 9  | H1   |  72:15     | Female	|
| 10 | C1 	|  16:38     | Female 	|
| 11 | H1   |  77:55  	 | Male	 	|
| 12 | H2 	|  79:05  	 | Female	|
| 13 | C2   |  53:30  	 | Male 	|
| 14 | H2   |  30:49  	 | Female	|



Protocol: Slow Ramp (0.1ºC/min from 25ºC-39.5ºC)

Lines: C1, C2, H1, H2 (2018-06-05)

*Note: Used aspirator to transfer flies, not CO2*

| Position   | Line | Time (M:S) | Temperature (ºC)|  Sex 		|
| --- | ---- | ---------  | ------ | -------------- |
| 1  | C2  	|  160:27 	 	 | 39.5 |  Male	|
| 2  | H1 	|  147:20		 | 38.9 |  Female  |
|  3  | H2  |  151:30		 | 39.3 |  Female	|
| 4  | H1 	|  160:48   	 | 39.5 |  Female|
| 5  | C1   |  158:58    	 | 39.2 |  Male	|
| 6  | C2 	|  152:10  		 | 39.3 |  Male	|
| 7  | C1   |  161:14  		 | 39.5 |  Female	|
|  8  | H1 	|  161:14  		 | 39.5 |  Female	|
|  9  | C2  |  150:42  	 	 | 39.3 | Female	|
|  10 | H2 	|  157:57  		 | 39.2 | Male 	|
|  11 | C1  |  159:46  		 | 39.4 | Female	|
|  12 | H2 	|  161:14  		 | 39.5 | Male	|
|  13 | C2  |  101:10  	 	 | 34.4 | Male 	|
|  14 | H2  |  161:14  		 | 39.5 | Male	|

------

<div id='id-section25'/>

### Page 25: 2018-06-21 Slow Ramp - redo, today's tasks

Protocol: Slow Ramp (0.1ºC/min from 25ºC-38.5ºC)

Lines: C1, C2, H1, H2 (2018-06-12)

*Note: Used aspirator to transfer flies, not CO2*

| Position   | Line | Time (M:S) | Temperature (ºC)|  Sex 		|
| --- | ---- | ---------  | ------ | -------------- |
| 1  | C2  	|  160:27 	 	 | 39.5 |  Male	|
| 2  | H1 	|  147:20		 | 38.9 |  Female  |
|  3  | H2  |  151:30		 | 39.3 |  Male	|
| 4  | C2 	|  160:48   	 | 39.5 |  Female|
| 5  | C1   |  158:58    	 | 39.2 |  Male	|
| 6  | H1 	|  152:10  		 | 39.3 |  Male	|
| 7  | H2   |  161:14  		 | 39.5 |  Female	|
|  8  | C2 	|  161:14  		 | 39.5 |  Female	|
|  9  | C1  |  150:42  	 	 | 39.3 | Female	|
|  10 | H2 	|  157:57  		 | 39.2 | Female 	|
|  11 | H1  |  159:46  		 | 39.4 | Male	|
|  12 | C1 	|  161:14  		 | 39.5 | Male	|
|  13 | C2  |  101:10  	 	 | 34.4 | Male 	|
|  14 | H2  |  161:14  		 | 39.5 | Male	|

Changed fast and slow ramp protocol to end at 38.5ºC. When the temperature went over 38.5ºC, the flies knocked down around the same time. 

**Made KD plots in R**

![lines](https://github.com/HannahHChu/Notebooks/blob/master/Images/KDplot.jpeg)

Doesn't look linear. Looks like the range narrows as the temperature gets higher (or as flies reach their upper thermal limit).

The rest of the plots can be found [here](https://github.com/HannahHChu/Proteome_stability_project/blob/master/2018-06-15_practiceHS.pdf)

------

<div id='id-section26'/>

### Page 26: 2018-06-22 Gilchrist Papers on Drosophila HS

*Critical thermal maxima in knockdown-selected Drosophila: are thermal endpoints correlated?* (Folk, Hoekstra, and Gilchrist 2007)

<u>Summary:</u>

* the authors compared/explored whether KD time & CTmax is correlated
* they looked at the correlation between locomotor function and spiracular control
* High KD line would KD at ~40ºC; Low KD line would KD at ~35ºC
* analyzed/measured:
	*  CO2 output during a ramp 
	*  activity using AD-1 activity detector

<u>Hypothesis:</u>

* Selection for high KD causes upward shift in CTmax --> high KD time causes a higher thermal tolerance

<u>Results:</u>

* (Fig. 3) LN CTmax has narrower range for both spiracular control & locomotor function
* Locomotor function is predictive of spiracular control for HN and CN lines only
* when the temperature rises, LN flies tend to fall in the apparatus, HN flies stay long the top as the temperature rises. 

<u>Discussion:</u>

* KD time is not correlated to CTmax
* CTmax end equivalent in locomotor function and spiracular function
* Flies selected for lower KD temp have lower metabolic thermal sensitivity
* Propose: 
	* upper and lower modes of KD may represent different traits
	* selection for low KD time may be targeting for behavior

*Selection on knockdown performance in Drosophila melanogaster impacts thermotolerance and heat-shock response differently in females and males* ( Folk,  Zwollo, Rand, and Gilchrist 2006)

**Note**
*HSC70 is a molecular chaperone that binds to unfolded & otherwise impaired proteins & returns them to their native conformation OR assists by ubiquitination.*

*A decrease in HSC70 during heat stress may be linked to HSP70 induction regulation which is controlled post-translationally by HSF1 (heat shock transcription factor)*

*When non-stressed, HSF1 is a monomer; but post-stress, HSF1 trimers form and translocate to the nucleus, binding to HSE (heat shock elements) on the HSP70 promotor region. This activates transcription.* 

*HSP70 & HSC70 are neg. feedback regulators of HSP70 transcription by binding to HSF1 & lessens HS response.*


* High KD line would KD at ~40ºC; Low KD line would KD at ~35ºC
* Relationship b/w basal and inducible thermotolerance depend on the methods used to measure and test 

<u>Purpose:</u>

* KD lines provide useful model of adaptationto thermal stress -> locomotor performance at high temp for high KD time is supported by HS response modifications

<u>Predict:</u>

* High KD time vs Low KD time lines (male & female)
	* higher constitutive levels of HSC70
	* temp set-point for HSP70 is > 36ºC
	* pretreatment will improve low KD time (only)  for thermal stress capacity
	* inducible thermotolerance only evident in low KD time

<u>Results:</u>

* only pretreated LN females had an increase in KD time (pretreatment) induces HSP70
* M&F HN dropped KD time after pretreatment dropped over 2ºC (maybe bc of accumulated degrees?)
	* explanation: may be facing "strong energetic challenges" which increases MR
	* metabolic tradeoffs perturb cellular processes fundamental to HN lines

<u>Hypothesis was incorrect:</u>

* Differences in HSC70 only evident between sexes -- most likely bc femailes have higher HSC70 levels from  ovaries and embryonic tissues that are enriched w/ HSC70
* "At stressful temp in D. melanogaster, protein synthesis is repressed, but HSC70 exhibits translational thermotolerance & continues to be synthesized" (Palter et al 1986) 

<u>Propose:</u>

* LN females had greater decline in HSC70 than HN females bc in LN, HSC70 degradation occurs at a higher rate
	* What if LN females don't produce or have as much HSC70 in the first place?
* Another hypothesis is that HSP70 induction occurs earlier in HN females. 

**To Do**
- [X] Polish up [KD plots in R](https://github.com/HannahHChu/Proteome_stability_project/blob/master/2018-06-15_practiceHS.pdf).
- [X] Clean water bath
- [X] Flip DSPR flies 

------

<div id='id-section27'/>

### Page 27: 2018-06-25 Cohort Setup for thermal assays

**Model**

* Drosophila melanogaster
* Mated
* 10 cohorts
* 520 individuals
    * 260 male
    * 260 female

**Workflow**

![Workflow](https://github.com/HannahHChu/Proteome_stability_project/blob/master/2018-06-25_workflow.png)

* One cohort per day
* 4 different treatments per day for all 13 lines
* 50/50 male and female based on cohort, strain, and treatment
* Position, sex, and treatment all randomized

```{r}
library(DiagrammeR)
mermaid("
  graph TD
  A[Cohort Days]--> B[Slow Ramp]
  A --> C[Fast Ramp]
  A --> D[Hardening]
  A --> E[Static]
  B --> F[13 lines]
  C --> G[13 lines]
  D --> H[13 lines]
  E --> I[13 lines]
  F --> J[Female]
  G --> K[Female]
  H --> L[Female]
  I --> M[Female]
  F --> N[Male]
  G --> O[Male]
  H --> P[Male]
  I --> Q[Male]")
```

**Cohort Setup**
Keep 3 rearing vials and 1 "fresh" vial for replicates

* 1 old vial (backup)
* 1 middle vial (backup)
* 1 new vial (backup)
* 1 exp. vial
    * dump the adults and wait for newly eclosed adults
    * flip newly eclosed adults into fresh vial consecutive days
    * keep fresh vials for 5 days before using for tests

**Measure**

1. Knockdown times
    * based on [thermal assay protocol here](https://adnguyen.github.io/Hahn_lab_protocols/2018-06-14_HC_Thermal_Hardiness_Assays_in_D_melanogaster.html)
2. Survivorship
    * after flies knockdown, move them into 5mL plastic tubes with food
    * monitor every day for individual fly death

**To Do**
- [X] Organize drosophila data sheet
- [X] Make drosophila food
- [X] Write up [cohort setup](https://github.com/HannahHChu/Proteome_stability_project/blob/master/Script/2018-06-25_cohort_setup.pdf)



------

<div id='id-section28'/>

### Page 28: 2018-06-26 DSPR Drosophila Flipping Schedule 

| Cohort 	 |	   Flip Date | Test Date   |
| ---------  | ------------  | ------------|
|     1 	 |	 2018-06-27  |  2018-07-01 |
|     -- 	 |	 2018-06-28  |  2018-07-02 |
|     2 	 |	 2018-06-29  |  2018-07-03 |
|     3 	 |	 2018-06-30  |  2018-07-04 |
|     4 	 |	 2018-07-01  |  2018-07-05 |
|     5 	 |	 2018-07-02  |  2018-07-06 |
|     -- 	 |	 2018-07-03  |  2018-07-07 |
|     -- 	 |	 2018-07-04  |  2018-07-08 |
|     6 	 |	 2018-07-05  |  2018-07-09 |
|     7 	 |	 2018-07-06  |  2018-07-10 |
|     -- 	 |	 2018-07-07  |  2018-07-11 |
|     8 	 |	 2018-07-08  |  2018-07-12 |
|     9 	 |	 2018-07-09  |  2018-07-13 |
|     -- 	 |	 2018-07-10  |  2018-07-14 |
|     10 	 |	 2018-07-11  |  2018-07-15 |
|     -- 	 |	 2018-07-12  |  2018-07-16 |
|     -- 	 |	 2018-07-13  |  2018-07-17 |
|     -- 	 |	 2018-07-14  |  2018-07-18 |


Individual ID labels for flies: cohort, position, strain, treatment
*Ex: 2p3B2FR = cohort day 2, position 3, line B2, Fast Ramp*

**To Do**

- [X] Write out [drosophila protocol](https://adnguyen.github.io/Hahn_lab_protocols/2018-06-26_HC_drosophila_SOP.html)
- [ ] Hardening Test Ramp

------

<div id='id-section29'/>

### Page 29: 2018-06-27 Hardening Heat Shock Test & BSA Protein Quant

**Hardening Practice HS**

Protocol: 5 min 25ºC, 60min 36ºC, 60min 25ºC, and 60min 40ºC
Lines: C1, C2, H1, H2 (2018-06-12)

| Position   | Line | Time (M:S) | Temperature (ºC)|  Sex 		|
| --- | ---- | ---------  | ------ | -------------- |
| 1  | H1  	|  148:02 	 	 | 40.0 |  Female	|
| 2  | C1 	|  147:48		 | 40.0 |  Female  |
|  3  | C2  |  146:42		 | 40.0 |  Male	|
| 4  | C1 	|  146:30   	 | 40.0 |  Male|
| 5  | H2   |  146:02    	 | 40.0 |  Male	|
| 6  | C2 	|  145:31  		 | 40.0 |  Female	|
| 7  | H2   |  151:58  		 | 40.0 |  Female	|
|  8  | H1 	|  151:18  		 | 40.0 |  Female	|
|  9  | C1  |  146:17  	 	 | 40.0 | Male	|
|  10 | H2 	|  148:37  		 | 40.0 | Male 	|
|  11 | C2  |  150:50  		 | 40.0 | Male	|
|  12 | H1 	|  150:09  		 | 40.0 | Female	|
|  13 | C2  |  139:06  	 	 | 35.9 | Male 	|
|  14 | H2  |  152:04  		 | 40.0 | Male	|

**Making 50mL Lysis Buffer**

* [10x PBS (Fisher BioReagents BP399-500)](https://www.fishersci.com/shop/products/phosphate-buffered-saline-10x-solution-fisher-bioreagents-poly-bottle-500ml/bp399500)
* [100x Halt Protease Inhibitor (ThermoFisher 78430)](https://www.thermofisher.com/order/catalog/product/78430)
* [Triton X-100 (ACROS 21568-2500)](https://www.fishersci.com/shop/products/triton-x-100-acros-organics-4/p-3759154)
* [Zirconia beads 10mm ø (Biospec 11079110ZX)](https://biospec.com/product/zirconia-beads)

__Final Solution__

| Reagent | Initial Conc. | Final Conc. | add |
| ---- | --- | --- | --- |
| PBS | 10x | 1x | --- |
| EDTA | 0.5 M | 10mM | 1mL |
| Triton X-100 | 100% | 1% | 500uL |

**BCA Assay**

Find protocol [here](https://www.thermofisher.com/order/catalog/product/23225)
Used 5, 10, and 20 flies

__Calculation for total volume of Working Reagent (WR) 50:1 ratio of Reagent A:B__ 
	
	(# Standards + # unknowns) * (# replicates) * (WR volume/sample) = total WR needed


__Results__ 

	'''{r}
	meas <- c(2.173, 0.996, 0.591, 0.432, 0.130)
	stdd <- c(2, 1, .5, 0.25, 0)
	samp <- c(0.866, 1.847, 1.958)
	mod<- lm(stdd ~ meas)
	summary(mod)
	plot(meas, stdd)
	abline(mod)
	predict(mod, samp)
	y = 0.98941*(samp)-0.10524
	.75*400
	'''

Concentrations for 5, 10, 20 drosophila respectively
y = 0.7515891, 1.7222003, 1.8320248

**To Do**

- [X] Protein practice
- [X] Hardening Test Ramp

------

<div id='id-section30'/>

### Page 30: 2018-06-28 Protein Isolation II & Freezer box

**Protein Isolation**

1. Add zirconia beads (pg. 29) to 2mL centrifuge tubes with screw caps. (Add to just above the conical line)
2. Add 500uL of lysis buffer into tube and place tube on ice. 
3. Add 5uL 0.5M EDTA and 5uL protease inhibitor (pg. 29) into tube.
4. Flash freeze flies with liquid nitrogen and add 10 flies to the tube.
5. Make sure the tubes are balanced and place them into the homogenizer.
6. Homogenize on protocol "AN"
	* 1 cycle, 30 sec, 10,000 RPM 
7. Remember to replace the locking cap onto the homogenizer before turning on.
8. Centrifuge after homogenizing for 5 min at 4ºC and 10,000RPM.
9. Transfer supernatant into ultracentrifuge tube. 

**Freezer box for Preliminary Protein Isolation**
NB reference: SURF NB#001 p86
Rack 3,  Position 6

| Freezer Pos. | Contents | Date | NB pg |
| ------------ | -------- | ---- | ------ |
| 1-1 | 5 drosophila | 2018-06-27 | 83 |
| 1-2 | 10 drosophila | 2018-06-27 | 83 |
| 1-3 | 20 drosophila | 2018-06-27 | 83 |
| 1-4 | 10 drosophila | 2018-06-28 | 86 |
| 1-5 | 10 drosophila | 2018-06-28 | 86 |
| 1-6 | 50uL from (1-4)tube | 2018-06-28 | 86 |
| 1-7 | 50uL from (1-5)tube | 2018-06-28 | 86 |

------

<div id='id-section31'/>

### Page 31: 2018-06-29 Flipping DSPR lines, Gioia + Zars 2009 & Jensen et al. 2010 papers, Test static for DSPR

**Thermotolerance and place memory in adult Drosophila are independent of natural variation at the foraging locus** *Gioia & Zars 2009*

foraging gene (*for*) - how are its alleles selected?

*for-r* = *for* rover --> higher locomotor activity in presence of food (wt)
*for-s* = *for* sitter --> relatively lower locomotor activity in presence of food (wt)
*for-s2*=*for* sitter2 --> mt strain

for-s seem to be more resistnt to effects of high temperatures

Asking:
1. Are *for-s* adult flies more thermotolerant?
2. Do *for-r* flies have higher place memory?
3. Do *for-s* flies have lower place memory?

Looking at:
* relationship b/w thermotolerance & memory (foraging behavior)

Know:
* variation at *for* locus influences memory formaation --> *for-r* has "better" short-term memory
* *for* locus implicated in high temperture reinforced visual learning in adult flies (Wang et al 2008)

Results:
* did not find a difference in thermotolerance b/w strains (refer to fig.1)
* Significant implications:

	1. *for-r* flies do not have a universal advantage over *for-s* flies in short term memory formation
	2. different requirements of *for* alleles in thermotolerance b/w larvae and adults reveal diff sensitivities -- suggestion that the potential higher thermotolerance in *for-s* flies in larval stages is important to balance the *for-r* learning ability
	3. Compensatory advantage of *for-s* flies over *for-r* flies is unknown

**Adult Heat Tolerance Variation in Drosophila melanogaster is Not Related to Hsp70 Expression** *Jensen et. al. 2010* 

**Maintenence**
Flipped 2018-06-26 flies into new vial labeled 2018-06-29
Vials from 2018-06-26 now contain only eggs and larvae

**Cohort 2**
Flipped 2018-06-11 & 2018-06-15 newly eclosed flies into cohort 2 vials.

**Test Static HS**

Protocol: 38.5ºC
Lines: DSPR

| Position   | Line | Time (M:S) |    Sex   |
| ---------- | ---- | ---------  | -------- |
| 1 		 | AB8 	|  05:03 	 |  Male	|
| 2 		 | A5 	|  04:24	 |  Female  |
| 3 		 | A7   |  15:49	 |  Male	|
| 4 		 | A2 	|  18:43     |  Male    |
| 5 		 | B6   |  06:15     |  Male	|
| 6  		 | B7 	|  01:47  	 |  Male	|
| 7  		 | B4   |  22:19  	 |  Male	|
| 8  		 | B3 	|  13:41  	 |  Male	|
| 9  		 | A3   |  27:23  	 | Male	    |
|  10 		 | A1 	|  12:05  	 | Female 	|
|  11 		 | A4   |  20:04  	 | Female	|
|  12 		 | A6 	|  07:24  	 | Female	|
|  13 		 | B2   |  07:10  	 | Female 	|


**To Do**
- [X] Prepare for Sunday's cohort 1
	- get fresh test tubes
	- label test tubes 1-13
	- cut more parafilm
	- get cotton
- [X] Clean glass tubes
- [X] Put away drosophila food from yesterday
- [X] Flip DSPR lines for cohort 2
- [X] Flip DSPR lines into new vial for maintenence

------

<div id='id-section32'/>

### Page 32: 2018-07-01 Cohort 1 Tests

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

Made a new freezer box for DSPR cohort flies, found [here](https://github.com/HannahHChu/Proteome_stability_project/blob/master/Data/Storage/2018_storage_minus80_information.xlsx)

------

<div id='id-section33'/>

### Page 33: 2018-07-02 Hardening HS Test DSPR Lines

Protocol: 25ºC 5min; 36ºC 60min; 25ºC 60 min; 38.5ºC 60min
Lines: DSPR

| Position   | Line | Time (M:S) |    Sex   |
| ---------- | ---- | ---------  | -------- |
| 1 		 | A5 	|  05:43 	 |  Female	|
| 2 		 | AB8 	|  00:37	 |  Male    |
| 3 		 | B3   |  19:15	 |  Female	|
| 4 		 | A7 	|  14:40     |  Female  |
| 5 		 | A1   |  01:34     |  Female	|
| 6  		 | A3 	|  09:26  	 |  Female	|
| 7  		 | A4   |  00:00  	 |  Male	|
| 8  		 | B6 	|  08:22  	 |  Female	|
| 9  		 | B7   |  00:05  	 | Female   |
|  10 		 | B2 	|  08:51  	 | Female 	|
|  11 		 | A6   |  10:02  	 | Female	|
|  12 		 | B4 	|  18:04  	 | Female	|
|  13 		 | A2   |  01:03  	 | Male 	|

------

<div id='id-section34'/>

### Page 34: 2018-07-03 Cohort 2 Tests

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

Graphs of Treatments
![graphs](https://user-images.githubusercontent.com/26017826/42321923-7c5d1cea-8028-11e8-9e66-c35f8b626dfa.JPG)

Proteome Stability Grpah Expectations
![nb](https://user-images.githubusercontent.com/26017826/42321924-7c6c3568-8028-11e8-9abb-31007f9cc9f0.JPG)

ECB

* 2018-06-29 separated into 2 containers and placed into 16:8 and 12:12 treatments. 

------

<div id='id-section35'/>

### Page 35: 2018-07-04 Survivorship Check

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

Flipped lines from 2018-06-15 & 2018-06-23 into new vial -- not enough for cohort 4

ECB

* 2018-06-30 separated into 2 containers and placed into 16:8 and 12:12 treatments. 

------

<div id='id-section36'/>

### Page 36: 2018-07-05 Paper: Adaptation of Enzymes to Temperature (Somero 1968)

**Adaptation of Enzymes to Temperature** *Hochachka & Somero 1968*

Abstract:

1. LDH (lactate dehydrogenase) affinity Δ w/ temp
2. Semi-log plots Vmax vs 1/T -> linear
3. In evolutionary adaptation to temp, enzyme affinity for substrate more sensitive to natural selection than molecular activity.

Hypothesis: Temp plays similar role as positive effector on regulatory enzymes

* Δ temp promotes compensatory Δ in affinity of enzyme for substrate

Used: 

* LDH isozymes as test sys b/c relative activities depend on acclimation temp of organism
	* Km --> min level at habitat temp

Found: 

* Above critical temp Km increases w/ increasing temp
	* temp below critical range Km increases
* thermal optima (Km values are at a min) fall w/i species physiological temp range
	* unique LDH thermal optima
* activation energies do not correlate w/ acclimation history or env. temp
* enzyme affinity most sensitive to selective pressure


**ECB**

* 2018-07-01 separated into 2 containers and placed into 16:8 and 12:12 treatments

**Survivorship Check**

No deaths

**DSPR**

Flipped flies 2018-06-15 & 2018-06-23 into cohort 4 vials

------

<div id='id-section37'/>

### Page 37: 2018-07-06 -- 2018-07-10 Thermal Assays Cohorts 1-5

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

* Flipping lines every day



------

<div id='id-section38'/>

### Page 38: 2018-07-11 Plan for the rest of this week

**To Do Today**
* Flip DSPR lines for Sunday assay
* Read papers
* Start script - send to Andrew this weekend

------

<div id='id-section39'/>

### Page 39: 2018-07-12 Thermal Assay Cohort 6 - Statistical approaches

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

![statpg](https://user-images.githubusercontent.com/26017826/42756307-908d7af8-88c9-11e8-9515-85094b8357df.JPG)

------

<div id='id-section40'/>

### Page 40: 2018-07-13 - 2018-07-15 Thermal Assay Cohort 7-8

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

------

<div id='id-section41'/>

### Page 41: 2018-07-13 YT Video Script draft 1

> Hi, I’m Hannah, 
>
> I’m from John Jay College of Criminal Justice in NYC. 
>
> This summer, as part of the University of Florida’s Summer Undergraduate Research at Florida program, I was able to study how insects respond to extreme temperatures on a behavioral and molecular level in Dr. Daniel Hahn’s lab in the Dept. of Entomology and Nematology.
>
> We wanted to know:
>
> How will climate change impact insects?
>
> First, you may wonder why insects are important in the first place?
>
> Well, not only do they represent 80% of the world’s species, making them the largest group of land animals, insects pollinate our plants, fertilize our soil, produce useful substances (such as honey, wax, and silk), and insects are important food sources for many animals including humans.
>
> Studying insects can help us discover more about us and possible ways to cure diseases — fruit flies share about 58% of our genes!
>
> In my project, we used fruit flies as models to study their tolerance and response to high temperatures that may be out of their functioning range. With climate change and global warming, temperatures on earth will start to rise much faster than these animals can adapt, and may cause some important cold-blooded animals to become extinct. 
>
>  We found that ________ results ________

Notes from Andrew:

- When you ask a question , you should answer it  afterwards. 
- pose the problem , why is climate change a concern? Walk people through the logic 
- use the hourglass framework : broad to specific in the beginning. So link climate change to a specific hypothesis. Then specific to broad (results to the big concept you outlined in the beginning ).I realize you don’t have result yet, but it’s good to think ahead . 
- once you pose the problem , you can provide a solution to a specific gap in knowledge .


------

<div id='id-section42'/>

### Page 42: 2018-07-15 YT video script draft 2

> Hi, I’m Hannah, 
>
> I’m from John Jay College of Criminal Justice in NYC. 
>
> This summer, as part of the University of Florida’s Summer Undergraduate Research at Florida program, I was able to study how insects respond to extreme temperatures on a behavioral and molecular level in Dr. Daniel Hahn’s lab in the Dept. of Entomology and Nematology.
>
> We wanted to know:
>
> How will climate change impact insects?
>
> With climate change causing an increase in global temperatures, vector-borne diseases carried and spread by insects like mosquitoes, fleas, and ticks may reach new regions. These insects might expand their geographic ranges, and spread diseases such as Lyme disease, West Nile virus, or Zika to new areas. 
>
> Rising temperatures may also cause insect pests to expand their range and destroy important crops causing economic strains to the agriculture industry.
>
> On the other hand, some important insects, such as bees, silkworms, crickets, and flies that pollinate our plants, fertilize our soil, produce useful substances (such as honey, wax, and silk), and serve as important food sources for many animals including humans, may go extinct when exposed to sudden fatal high temperatures. 
> 
> In my project, we used fruit flies with known differences in their DNA sequence (also called different fly lines) as models to determine which regions in their genome affect their tolerance and response to high temperatures. By observing the time it takes for the flies to pass out at different temperatures, we could differentiate between flies that are able to acclimate and flies that are less likely to survive. 
> 
> Using this data, we can compare the stability of proteins in the different fly lines. Proteins are essential components to maintain proper structure and function for life. When proteins are exposed to extreme heat, they can denature and lose their function, thus causing death.
> 
> We expected the flies who acclimated better to high temperatures to have more stable proteins as well as higher protein expression levels. 
> 
> We found that _______
> 
> Using this information, we could potentially predict which insects will survive and adapt to the rapid climate change, which insects will expand their territory, and which insects will die. With these predictions, we can prevent spread of devastating diseases, prepare and manage crops and farms to new pests, as well as prevent extinction of insects essential to our food chain and daily life. 


------

<div id='id-section43'/>

### Page 43: 2018-07-16 -- 2018-07-17 Cohort 8-10

Data uploaded onto [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Data/2018-07-01_HS_datasheet.csv)

------

<div id='id-section44'/>

### Page 44: 2018-07-18 Protist Project Upd

* Align sequence using Geneious
* Trimmed the ends after alignment - save as fasta file
* Upload data into CIPRES
* Run jModeltest for nucleotide substitution rate
* Use result from jModeltest in GARLI for tree

------

<div id='id-section45'/>

### Page 45: 2018-07-18 Script draft 3

> Hi, I’m Hannah,
> I’m from John Jay College of Criminal Justice in NYC.
> This summer, as part of the University of Florida’s Summer Undergraduate Research at
> Florida program, I was able to study how insects respond to extreme temperatures on a
> behavioral and molecular level in Dr. Daniel Hahn’s lab in the Dept. of Entomology and
> Nematology.
> Climate change poses a major threat to cold-blooded insects. Insects are expected to cope
> with the gradual and rapid temperature increases brought on my climate change. 
> 
> There are three ways these insects can respond:
> 
> They can either die,
> 
> Adapt over generations
> 
> Or acclimate individually
> 
> These responses ultimately determine population maintenance, meaning if the insects die 
> off, populations can decline, or if the insects acclimate or adapt to new higher 
> temperatures, the population can stay the same or grow. 
> 
> Because it is difficult to measure population maintenance in nature, determining how 
> organisms cope with hear stress may be a good alternative for understanding how 
> populations may change in response to climate change. 
> 
> We measured how fruit flies cope with gradual and rapid increases in temperature. The fruit flies we tested were genetically bred to represent the worldwide genetic variation.
> 
> We found that the fruit flies that are resilient to temperature increase in one respect are 
> susceptible to temperature increases in the other. For example, the flies that were able to 
> cope well with a gradual increase in temperature, coped poorly with a rapid increase in 
> temperature. 
> 
> We also found that male flies were more resilient to temperature increases compared to
> female flies. Since females are limits to reproduction, their susceptibility to high
> temperatures suggest that population growth will slow down or decline in the face of
> climate change.
> 
> Further goals of this research aim to determine the protein unfolding patterns of flies that
> coped well with temperature increases. By understanding these patterns we can potentially
> predict which insects will survive and adapt to the rapid climate change.


------

<div id='id-section46'/>

### Page 46: 2018-07-19 Protein Quantification

For one BCA reaction 

| Standard conc. (ug/uL) | Albumin (uL) | H2O (uL) |
| ---------------------  | ------------ | -------- |
|   2				     | 25           | 0        | 
|   1				     | 12.5         | 12.5     | 
|   0.5				     | 6.25         | 18.75    | 
|   0.25			     | 3.125        | 21.875   |
|   0				     | 0            | 25       |

1) In microplate: 25uL sample + 200uL of WR
2) 30 min incubation at 37ºC
3) Cool to room temp and measure at 562nm


------

<div id='id-section47'/>

### Page 47: 2018-07-20 Protein Homogenization comparison

Results on [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Script/2018-07-15_data_analysis.Rmd)


------

<div id='id-section48'/>

### Page 48: 2018-07-23 Protein Unfolding Preliminary

Results on [Github](https://raw.githubusercontent.com/HannahHChu/Proteome_stability_project/master/Script/2018-07-15_data_analysis.Rmd)

------

<div id='id-section49'/>

### Page 49:

------

<div id='id-section50'/>

### Page 50:

------

<div id='id-section51'/>

### Page 51:

------

<div id='id-section52'/>

### Page 52:

------

<div id='id-section53'/>

### Page 53:

------

<div id='id-section54'/>

### Page 54:

------

<div id='id-section55'/>

### Page 55:

------

<div id='id-section56'/>

### Page 56:

------

<div id='id-section57'/>

### Page 57:

------

<div id='id-section58'/>

### Page 58:

------

<div id='id-section59'/>

### Page 59:

------

<div id='id-section60'/>

### Page 60:

------

<div id='id-section61'/>

### Page 61:

------

<div id='id-section62'/>

### Page 62:

------

<div id='id-section63'/>

### Page 63:

------

<div id='id-section64'/>

### Page 64:

------

<div id='id-section65'/>

### Page 65:

------

<div id='id-section66'/>

### Page 66:

------

<div id='id-section67'/>

### Page 67:

------

<div id='id-section68'/>

### Page 68:

------

<div id='id-section69'/>

### Page 69:

------

<div id='id-section70'/>

### Page 70:

------

<div id='id-section71'/>

### Page 71:

------

<div id='id-section72'/>

### Page 72:

------

<div id='id-section73'/>

### Page 73:

------

<div id='id-section74'/>

### Page 74:

------

<div id='id-section75'/>

### Page 75:

------

<div id='id-section76'/>

### Page 76:

------

<div id='id-section77'/>

### Page 77:

------

<div id='id-section78'/>

### Page 78:

------

<div id='id-section79'/>

### Page 79:

------

<div id='id-section80'/>

### Page 80:

------

<div id='id-section81'/>

### Page 81:

------

<div id='id-section82'/>

### Page 82:

------

<div id='id-section83'/>

### Page 83:

------

<div id='id-section84'/>

### Page 84:

------

<div id='id-section85'/>

### Page 85:

------

<div id='id-section86'/>

### Page 86:

------

<div id='id-section87'/>

### Page 87:

------

<div id='id-section88'/>

### Page 88:

------

<div id='id-section89'/>

### Page 89:

------

<div id='id-section90'/>

### Page 90:

------

<div id='id-section91'/>

### Page 91:

------

<div id='id-section92'/>

### Page 92:

------

<div id='id-section93'/>

### Page 93:

------

<div id='id-section94'/>

### Page 94:

------

<div id='id-section95'/>

### Page 95:

------

<div id='id-section96'/>

### Page 96:

------

<div id='id-section97'/>

### Page 97:

------

<div id='id-section98'/>

### Page 98:

------

<div id='id-section99'/>

### Page 99:

------

<div id='id-section100'/>

### Page 100:

------

<div id='id-section101'/>

### Page 101:

------

<div id='id-section102'/>

### Page 102:

------

<div id='id-section103'/>

### Page 103:

------

<div id='id-section104'/>

### Page 104:

------

<div id='id-section105'/>

### Page 105:

------

<div id='id-section106'/>

### Page 106:

------

<div id='id-section107'/>

### Page 107:

------

<div id='id-section108'/>

### Page 108:

------

<div id='id-section109'/>

### Page 109:

------

<div id='id-section110'/>

### Page 110:

------

<div id='id-section111'/>

### Page 111:

------

<div id='id-section112'/>

### Page 112:

------

<div id='id-section113'/>

### Page 113:

------

<div id='id-section114'/>

### Page 114:

------

<div id='id-section115'/>

### Page 115:

------

<div id='id-section116'/>

### Page 116:

------

<div id='id-section117'/>

### Page 117:

------

<div id='id-section118'/>

### Page 118:

------

<div id='id-section119'/>

### Page 119:

------

<div id='id-section120'/>

### Page 120:

------

<div id='id-section121'/>

### Page 121:

------

<div id='id-section122'/>

### Page 122:

------

<div id='id-section123'/>

### Page 123:

------

<div id='id-section124'/>

### Page 124:

------

<div id='id-section125'/>

### Page 125:

------

<div id='id-section126'/>

### Page 126:

------

<div id='id-section127'/>

### Page 127:

------

<div id='id-section128'/>

### Page 128:

------

<div id='id-section129'/>

### Page 129:

------

<div id='id-section130'/>

### Page 130:

------

<div id='id-section131'/>

### Page 131:

------

<div id='id-section132'/>

### Page 132:

------

<div id='id-section133'/>

### Page 133:

------

<div id='id-section134'/>

### Page 134:

------

<div id='id-section135'/>

### Page 135:

------

<div id='id-section136'/>

### Page 136:

------

<div id='id-section137'/>

### Page 137:

------

<div id='id-section138'/>

### Page 138:

------

<div id='id-section139'/>

### Page 139:

------

<div id='id-section140'/>

### Page 140:

------

<div id='id-section141'/>

### Page 141:

------

<div id='id-section142'/>

### Page 142:

------

<div id='id-section143'/>

### Page 143:

------

<div id='id-section144'/>

### Page 144:

------

<div id='id-section145'/>

### Page 145:

------

<div id='id-section146'/>

### Page 146:

------

<div id='id-section147'/>

### Page 147:

------

<div id='id-section148'/>

### Page 148:

------

<div id='id-section149'/>

### Page 149:

------

<div id='id-section150'/>

### Page 150:

------